### ISSUE TITLE
Some tags don't work?

### ISSUE THREAD

**Issue by mglaman** [mglaman/contribkanban.com#52]
Sprint board: https://contribkanban.com/sprint/Performance
D.o Search: https://www.drupal.org/project/issues/search?issue_tags=Performance
Term page: https://www.drupal.org/taxonomy/term/151165

wut.

**Comment by tloudon** [comment]
@mglaman try to curl https://www.drupal.org/api-d7/taxonomy_term.json?name=Performance
here's a pretty print of the response:
```
{"self"=>"https://www.drupal.org/api-d7/taxonomy_term?name=Performance",
 "first"=>"https://www.drupal.org/api-d7/taxonomy_term?name=Performance&page=0",
 "last"=>"https://www.drupal.org/api-d7/taxonomy_term?name=Performance&page=0",
 "list"=>
  [{"tid"=>"132",
    "name"=>"Performance",
    "description"=>"",
    "weight"=>"0",
    "node_count"=>10,
    "url"=>"https://www.drupal.org/project/issues/search?issue_tags=Performance",
    "parent"=>[],
    "parents_all"=>[{"uri"=>"https://www.drupal.org/api-d7/taxonomy_term/132", "id"=>"132", "resource"=>"taxonomy_term"}]},
   {"tid"=>"32238",
    "name"=>"performance",
    "description"=>"",
    "weight"=>"0",
    "node_count"=>10,
    "url"=>"https://www.drupal.org/taxonomy/term/32238",
    "parent"=>[],
    "parents_all"=>[{"uri"=>"https://www.drupal.org/api-d7/taxonomy_term/32238", "id"=>"32238", "resource"=>"taxonomy_term"}]},
   {"tid"=>"151165",
    "name"=>"Performance",
    "description"=>"",
    "weight"=>"0",
    "node_count"=>0,
    "url"=>"https://www.drupal.org/taxonomy/term/151165",
    "parent"=>[],
    "parents_all"=>[{"uri"=>"https://www.drupal.org/api-d7/taxonomy_term/151165", "id"=>"151165", "resource"=>"taxonomy_term"}]}]}
```
TL;DR there are multiple taxo terms and it looks to me like the best path forward is to do some kind of gross substring match on each Term.url to see if it matches 'https://www.drupal.org/project/issues/search?issue_tags=' or whatevs.  Maybe ping a d.o infrastructure member??

**Comment by tloudon** [comment]
^ https://www.drupal.org/taxonomy/term/132

**Comment by mglaman** [comment]
So I grab the first entry. I'm just a jackwagon and sorted by weight instead of tid, because that made sense for some reason https://github.com/mglaman/contribkanban.com/blob/master/app/scripts/controllers/browseSprintCtrl.js#L45
